parsl/monitoring/db_manager.py

```diff
@@ -126,29 +128,50 @@ class Task(Base):
         __tablename__ = TASK
         task_id = Column('task_id', Integer, nullable=False)
         run_id = Column('run_id', Text, nullable=False)
-        hostname = Column('hostname', Text, nullable=True)
         task_depends = Column('task_depends', Text, nullable=True)
-        task_executor = Column('task_executor', Text, nullable=False)
         task_func_name = Column('task_func_name', Text, nullable=False)
-        task_time_submitted = Column(
-            'task_time_submitted', DateTime, nullable=True)
-        task_time_running = Column(
-            'task_time_running', DateTime, nullable=True)
-        task_time_returned = Column(
-            'task_time_returned', DateTime, nullable=True)
         task_memoize = Column('task_memoize', Text, nullable=False)
         task_hashsum = Column('task_hashsum', Text, nullable=True)
         task_inputs = Column('task_inputs', Text, nullable=True)
         task_outputs = Column('task_outputs', Text, nullable=True)
         task_stdin = Column('task_stdin', Text, nullable=True)
         task_stdout = Column('task_stdout', Text, nullable=True)
         task_stderr = Column('task_stderr', Text, nullable=True)
+
+        task_time_returned = Column(
+            'task_time_returned', DateTime, nullable=True)
```

Comment:
I feel it may be strange to users that this table only has `task_time_returned` without `task_time_submitted`.

It may look better to either add `task_time_submitted` (indicating the submit time of the first try or the time loaded from cache for no try case) or remove both `task_time_submitted` and `task_time_returned` to make it a pure task metadata table.

Both `task_time_submitted` and `task_time_returned` can be inferred from `try` table for those tasks that are not cached.

Reply:
I think both tasks and tries should have a submitted and returned time:

* the task submitted/returned times should be (approximately) the time the workflow calls an app that creates the task, to the time that the app invocation's future has a result available.

* the try time submitted should be (approximately) the time that the DFK makes a submit call to an executor, to the time that the executor returns a value in the corresponding executor future.

I will make these changes.

Reply:
@ZhuozhaoLi I reworked the timing columns in the database.

In the `task` table there are now timestamps for:
  i) when the relevant app was invoked
 ii) when the task returned.

In try `try` table there are now timestamps for:
  i) when the try was submitted to an executor
 ii) when the try started running on a worker
iii) when the try completed